frontend/src/constants/routes.js

```diff
@@ -21,6 +21,7 @@ export const INTERVIEW_SCHEDULING = '/interview-scheduling';
 export const FORBIDDEN = '/403';
 export const NOT_FOUND = '/404';
 export const EVENT_SIGN_IN_OPTIONS = '/events/:id/signinoptions';
+export const INDUCTEE_SIGN_UP_FORM = '/inductee-signup-form'

```

Comment:
Should be called `INDUCTEE_SIGN_UP_PAGE` and `inductee-signup-page` since we don't navigate to a form, we navigate to a page